src/end_entity.rs

```diff
@@ -203,14 +203,14 @@ mod tests {
             // common name value, rather than `UTF8String`.
             params.distinguished_name.push(
                 rcgen::DnType::CommonName,
-                rcgen::DnValue::PrintableString("example.com".to_string()),
+                rcgen::DnValue::PrintableString(
+                    rcgen::PrintableString::try_from("example.com").unwrap(),
+                ),
             );
-            let cert = rcgen::Certificate::from_params(params)
+            let cert = params
+                .signed_by(&test_utils::make_keypair(), &issuer.cert, &issuer.key_pair)
                 .expect("failed to make ee cert (this is a test bug)");
-            let bytes = cert
-                .serialize_der_with_signer(&issuer)
-                .expect("failed to serialize signed ee cert (this is a test bug)");
-            CertificateDer::from(bytes)
+            cert.der().clone()
```

Comment:
`cert.into()`?

Reply:
I was able to avoid both in this case by binding the `Certificate` and just passing `ee_cert.der()` right into `expect_dns_name`.